### ISSUE TITLE
Unexpected Redirect

### ISSUE THREAD

**Issue by nikitashekhov** [mimshins/figma-backup#13]
Good day.

I started using your tool and every time I start `figma-backup-interactive` I get Unexpected Redirect error. 

Probably, you need to handle the redirect in the code.

Traces:
```
? Enter the ids of your figma projects:
(Separate the ids with SPACE(s). i.e. ID1 ID2 ID3) XXXXX (I entered just one)

? Enter the download timeout (in minutes):
(This number indicates the maximum amount of time the bot has to wait for a file to be downloaded.) 5

? Enter the interaction delay (in seconds):
(This number indicates the delay between interactions.) 2

? Enter the typing delay (in miliseconds):
(This number indicates the delay to type a new character.) 100

 Starting the backup task...
 Authenticating the bot...
 Stopping the bot...
/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:145
            throw new errors_1.BackupError(e);
                  ^
BackupError: Backup failed | Authorization failed | Unexpectedly redirected to "https://www.figma.com/files/ID/recent?fuid=FUID"
    at Bot._backupProjects (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:145:19)
    at async Bot.start (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:167:9)
    at async startBot (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/index.interactive.js:94:5)
    at async run (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/index.interactive.js:105:5) {
  _error: AuthorizationError: Authorization failed | Unexpectedly redirected to "https://www.figma.com/files/ID/recent?fuid=FUID"
      at Bot._login (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:66:19)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async Bot._authenticate (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:71:9)
      at async Bot._backupProjects (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:139:13)
      at async Bot.start (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:167:9)
      at async startBot (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/index.interactive.js:94:5)
      at async run (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/index.interactive.js:105:5) {
    _error: Error: Unexpectedly redirected to "https://www.figma.com/files/ID/recent?fuid=FUID"
        at new AuthorizationError (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/errors/AuthorizationError.js:7:51)
        at Bot._login (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:66:19)
        at runMicrotasks (<anonymous>)
        at processTicksAndRejections (node:internal/process/task_queues:96:5)
        at async Bot._authenticate (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:71:9)
        at async Bot._backupProjects (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:139:13)
        at async Bot.start (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:167:9)
        at async startBot (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/index.interactive.js:94:5)
        at async run (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/index.interactive.js:105:5)
  }
}
```

Mac OS 13.0.1, npm 8.11.0, figma-backup 2.0.0

**Comment by mimshins** [comment]
Hi, @nikitashekhov 
Can you try to backup using the `figma-backup` (the legacy way) and report me the results, please?

**Comment by nikitashekhov** [comment]
@mimshins 

Yep. I got this:
```
BackupError: Backup failed | Authorization failed | No element found for selector: form#auth-view-page > input[name="password"]
    at Bot._backupProjects (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:145:19)
    at async Bot.start (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:167:9)
    at async Object.handler (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/index.js:75:5) {
  _error: AuthorizationError: Authorization failed | No element found for selector: form#auth-view-page > input[name="password"]
      at Bot._login (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:50:19)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async Bot._authenticate (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:71:9)
      at async Bot._backupProjects (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:139:13)
      at async Bot.start (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:167:9)
      at async Object.handler (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/index.js:75:5) {
    _error: Error: No element found for selector: form#auth-view-page > input[name="password"]
        at assert (/Users/username/.npm-global/lib/node_modules/figma-backup/node_modules/puppeteer/lib/cjs/puppeteer/util/assert.js:28:15)
        at IsolatedWorld.click (/Users/username/.npm-global/lib/node_modules/figma-backup/node_modules/puppeteer/lib/cjs/puppeteer/common/IsolatedWorld.js:239:32)
        at processTicksAndRejections (node:internal/process/task_queues:96:5)
        at async submitLoginForm (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/utils/submitLoginForm.js:17:5)
        at async Promise.all (index 1)
        at async waitAndNavigate (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/utils/waitAndNavigate.js:4:5)
        at async Bot._login (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:43:13)
        at async Bot._authenticate (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:71:9)
        at async Bot._backupProjects (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:139:13)
        at async Bot.start (/Users/username/.npm-global/lib/node_modules/figma-backup/lib/Bot.js:167:9)
  }
}
```

**Comment by mimshins** [comment]
That's weird :/
Can I have the full command-line output?

**Comment by nikitashekhov** [comment]
It is full, as before I just put my credentials.

Does it work with Figma Organisation?

**Comment by flosob** [comment]
Hey Guys,

We have the same problem. We use an Figma Enterprise Account with multiple Workspaces. The structure here is Workspace/Teams/Projects/Files.

**Comment by mimshins** [comment]
Unfortunately, I didn't have time to investigate this further.
I'll make sure to check it out as soon as possible.

@flosob 
Please provide your command line output.

**Comment by mimshins** [comment]
@nikitashekhov 
I can't re-produce the issue myself,
Please re-run the command and opt-in the `--debug` flag. This way you can observe the bot and help me with the issue.

**Comment by mimshins** [comment]
@nikitashekhov @flosob 
This should be resolved (upgrade to v2.0.1), please confirm so I can close the thread.
